### PR TITLE
New job os-brick-ceph

### PR DESCRIPTION
**Pull request by stefanlupsa**

### CODE REVIEW

File: tasks/windows/configure-ceph-wnbd.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      win_template:
+        src: "windows/ceph-wnbd.conf"
+        dest: "C:\\ProgramData\\Ceph\\ceph.conf"
+      tags: configure-ceph-nbd
+
+    - name: Check ceph-rbd service exists to verify instalation
+      win_service:
+        name: "ceph-rbd"
+        state: started
+      tags: configure-ceph-nbd
```
Comment (petrutlucian94) [inline tasks/windows/configure-ceph-wnbd.yaml:55]: We should probably check the Ceph cluster connection by running a "rbd ls --pool volumes".

File: job_vars/os-brick-ceph.yaml [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+    - s-proxy
+    - s-object
+    - s-container
+    - s-account
+    - heat
+    - h-api
+    - h-api-cfn
+    - h-api-cw
+    - h-eng
+    - ceilometer-acentral
```
Comment (petrutlucian94) [inline job_vars/os-brick-ceph.yaml:72]: Here's a list of services that can be disabled:
* ceilometer
* heat
* q-metering and q-vpn

I think swift and c-bak can also be skipped.

File: job_vars/os-brick-ceph.yaml [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+win_python_archive_url: http://10.100.0.9/ci/python37.zip
+
+vmswitches:
+  - name: "{{ data_bridge }}"
+    subnet: "{{ data_network }}"
+    management: False
+
+volume_driver: cinder.volume.drivers.rbd.RBDDriver
+
+ceph_wnbd_msi_url: http://10.107.8.204/ceph_v16_0_0_beta.msi
```
Comment (petrutlucian94) [inline job_vars/os-brick-ceph.yaml:296]: Ok, so this is the URL that will have to be replaced.

File: job_vars/os-brick-ceph.yaml [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+    value: 'simplejson===3.13.2'
+
+win_python_packages:
+  - pywin32
+  - pbr
+  - pymysql
+  - ovs
+
+win_cherry_picks:
+  - project: openstack/neutron
```
Comment (petrutlucian94) [inline job_vars/os-brick-ceph.yaml:257]: This one has already merged: https://review.opendev.org/c/openstack/neutron/+/763159

**Review by petrutlucian94 — COMMENTED** [review]
Looks good, just a few minor suggestions.

File: tasks/windows/configure-ceph-wnbd.yaml (hunk 1 above)
Comment (stefanlupsa) [reply tasks/windows/configure-ceph-wnbd.yaml:55]: Will add the check in post-stack as this needs to happen after copying the keyring

File: job_vars/cinder-iscsi.yaml [hunk 5]
```diff
@@ -292,11 +292,11 @@ services:
     template: cinder-backup.conf
 
 
-win_cherry_picks:
-  - project: openstack/neutron
-    path: "{{ win_dir.build }}\\neutron"
-    patches:
-      - refs/changes/59/763159/1
+# win_cherry_picks:
+  # - project: openstack/neutron
```
Comment (petrutlucian94) [inline job_vars/cinder-iscsi.yaml:296]: Nit: we might as well remove this entry, keeping just the compute-hyperv one as a sample.

**Review by petrutlucian94 — APPROVED** [review]
Looks good, thanks for taking care of this!

File: job_vars/cinder-iscsi.yaml (hunk 5 above)
Comment (stefanlupsa) [reply job_vars/cinder-iscsi.yaml]: cleaned all outdated cherry picks, still have sample in the os-brick-fc.yaml

**Review by ociuhandu — APPROVED** [review]

**Pull request merged**